modules/images/auto-sizes/load.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
```

Comment:
Can you also add entries for this module to `.github/CODEOWNERS` like there are for the other modules?

Reply:
[a7dffc5](https://github.com/WordPress/performance/pull/904/commits/a7dffc584250ddaa67666cb6617eedb173ba1f37)